addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -596,7 +595,6 @@
                                 We build great products to solve your business problems.
                             </p>
                             <div class="s_btn pt18" data-name="Button">
-                                <a href="/aboutus" class="btn btn-primary">About us</a>
```

Comment:
does it make sense to have s_btn empty ? Can we jsut delete all the node.
Same for others places ;)

Reply:
Especially since I just merged the s_btn stuff entirely with FJA's branch, you will have conflict when rebasing on master once we forward-port.